crates/edr_napi/Cargo.toml

```
@@ -7,7 +7,9 @@ edition = "2021"
 crate-type = ["cdylib"]
 
 [dependencies]
+alloy-dyn-abi = { version = "0.7.6", default-features = false }
 alloy-sol-types = { version = "0.5.1", default-features = false, features = ["std"] }
+alloy-json-abi = { version = "0.7.4", default-features = false }
```

```suggestion
alloy-json-abi = { version = "0.7.4", default-features = false }
 alloy-sol-types = { version = "0.5.1", default-features = false, features = ["std"] }

```
Not fully alphabetical yet